### PR TITLE
Bug 1241399 - [TV][2.5][Web Apps] First time opening tutorial, background image will late shown

### PR DESCRIPTION
**Pull request by josyeh**

### CODE REVIEW

File: src/media/css/tutorial.styl [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
     transition: opacity 1s;
 
+    &.invisible {
+        opacity: 0;
```
Comment (spasovski) [inline src/media/css/tutorial.styl:31]: Might be smart to add a transition so it looks like the intended effect. I'd also recommend a spinner to let the user know something is loading, though in the screenshot for the bug it seems like there might already be one.

File: src/media/js/views/tutorial.js [hunk 2]
```diff
@@ -3,9 +3,28 @@ define('views/tutorial',
...
+    function loadBackgroundImage($slide, callback) {
+        imageLoader.src = $slide.find('.slide-image')
+                                .css('background-image')
+                                .replace(/url\(['"]?(.*?)['"]?\)/i, '$1');
+
+        imageLoader.onload = function () {
+            $slide.removeClass('invisible');
+
+            callback();
+        };
```
Comment (spasovski) [inline src/media/js/views/tutorial.js:18]: We should add an `onerror` and pretty much run the same function as the `onload` - or create a background CSS gradient as a filler. If the image 404s or something right now the content will never show up.

**Comment by spasovski** [comment]
r+wc

File: src/media/js/views/tutorial.js (hunk 2 above)
Comment (josyeh) [inline src/media/js/views/tutorial.js:18]: It looks weird for this tutorial if the image did not show up, so I would rather redirect the user back to the homepage when `onerror` happened.

**Pull request merged**